src/components/Navbar/NavBar.styles.js

```diff
@@ -1,13 +1,20 @@
 import styled from '@emotion/styled';
 import { Link } from 'react-router-dom';
 
+// If you change the height of this element, go to:
```

Comment:
I see that you marked the conversation about this comment being removed as resolved, but I don't understand how it is resolved? Did you agree to leave the comment?